boot.go

```diff
@@ -34,7 +34,7 @@ func init() {
 }
 
 func main() {
-	if os.Getenv("DEBUG") == "true" {
+	if os.Getenv("DEIS_DEBUG") == "true" {
```

Comment:
we can do that if we want but that would mean writing the same code for two different commands and currently the `server` command doesn't care about the gitreceive/conf where our debug flag is